packages/amplify-graphql-auth-transformer/src/utils/index.ts

```diff
@@ -56,6 +65,30 @@ export const ensureAuthRuleDefaults = (rules: AuthRule[]) => {
   }
 };
 
+export const getAuthDirectiveRules = (authDir: DirectiveWrapper): AuthRule[] => {
+  const rules = authDir.getArguments<{ rules: Array<AuthRule> }>({ rules: [] }).rules;
+
+  for (const rule of rules) {
+    const operations: (ModelOperation | 'read')[] = rule.operations ?? [];
```

Comment:
Is `rule.operations` expected to exist on all rules after this function runs? If so, it looks like that is not currently the case.

Reply:
no, discussed offline